ui-common/src/hooks/useWS.tsx

```diff
@@ -7,7 +7,7 @@ enum WebSocketReadyState {
   CLOSED
 }
 
-export const DEFAULT_QUERY = "leftOff(-1)";
+export const DEFAULT_QUERY = `leftOff("latest")`;
```

Comment:
where it is used? and why not use it in (https://github.com/up9inc/mizu/pull/984/files#diff-d6919ca03cedf72249cdb293e45ab5013085888488398f8a38da8baaef683c68R120)

Reply:
https://github.com/up9inc/mizu/commit/93725264c34f1df6b81fd31eaa5dde56194ea5ab